code/modules/power/supermatter/supermatter.dm

```diff
@@ -467,6 +467,7 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 		if(FLUX_ANOMALY)
 			var/obj/effect/anomaly/flux/flux = new(local_turf, has_changed_lifespan ? rand(250, 350) : null, FALSE)
 			flux.explosive = !has_changed_lifespan
+			flux.reduced_explosion = TRUE
```

Comment:
and while I'm here, what the- the supermatter creates the flux telling it not to be explosive and then tells it to possibly be explosive after? Fix this too by instead of doing a varset after, just make what you pass in as args this condition.
Basically, instead of

```dm
var/thing = new(_var = FALSE)
thing.var = condition
```

do
```dm
var/thing = new(_var = condition)
```
same deal with reduced explosion, which is why i'm asking you to fix `.explosive` in the first place
(yes _var is a bad name that goes against our naming policy but I can only ask for so much cleanup, if you rename that initial arg on flux kudos to you)